lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -63,6 +63,19 @@ public function __construct(array $params, $username, $password, array $driverOp
 
         $this->_conn = mysqli_init();
 
+        if (isset($driverOptions['ssl_key']) &&
+            isset($driverOptions['ssl_cert']) &&
+            isset($driverOptions['ssl_ca'])
```

Comment:
@pgrau agreed. MySQL documentation also states that the `ca` is only mandatory if also used on the server side so we have to make it optional. Making `ssl_key` and `ssl_cert` required is a good choice then.